server/src/utils/fight/fightMethods.ts

```diff
@@ -981,30 +1010,36 @@ const activateSuper = (
         // Change pet owner
         pet.master = fighter.id;
         pet.team = fighter.team;
+        // Lower pet initiative
+        pet.initiative -= 1;
       }
 
-      // Abort if no pet hypnotised
-      if (hypnotisedPets.length === 0) return false;
+      // Hypnotize opponent's brutes and bosses
+      const opponents = getOpponents({ fightData, fighter, bruteAndBossOnly: true})
+      opponents.forEach((opponent) => {
+        opponent.hypnotized = true;
+      });
 
       // Add hypnotise step
       fightData.steps.push({
         a: StepType.Hypnotise,
         b: fighter.index,
+        t: opponents.map((opponent) => opponent.index),
         p: hypnotisedPets,
       });
 
       break;
     }
     case SkillName.flashFlood: {
       // Choose opponent
-      const opponent = getRandomOpponent({ fightData, fighter, bruteOnly: true });
+      const opponent = getRandomOpponent({ fightData, fighter, bruteAndBossOnly: true });
 
       if (!opponent) {
         return false;
       }
 
-      // Drop shield
-      if (fighter.shield) {
+      // Drop shield if this is the last flashFlood cast
+      if (fighter.shield && fighter.weapons.length + (fighter.activeWeapon ? 1 : 0) < 6) {
```

Comment:
I don't understand the logic here.
Why would a brute throw away their shield, after throwing most of their weapons?

I'd rather throw it away anytime, or not at all.
Another solution would be to throw it as a weapon and deal damage from it, depending on block chance, but more complex to implement.
The last solution would make the most sense

Reply:
Like I said it only skips a 0.1 or so animation that does nothing on pets after they already moved, but yeah it almost doesn't change anything I'll remove it for safety.

From my discussion with players, their are 2 types of FlashFlood builds, the 12 weapon build, that wants to survive (they want to keep shield), and the 4-5 Big weapon build, that wants to survive until they can cast one time and mostly win the game (they want to drop shield to increase damage).

The idea is to drop shield as an All-In process, but when it's not an all-in, not drop it.

Throwing it is a nice idea, I will implement throwing shield to stun opponent in next commit if you are ok, but I still believe throwing it at last cast is both the most roleplay and the best for players of both FlashFlood builds.

Reply:
Or maybe you are right, well idk it's hard to tell

Reply:
Yes, throwing it at the opponent at the end of the last cast is ok, it just didn't make sense to throw it away (not at the opponent) at the end